lists/serviceList.h

```diff
@@ -0,0 +1,21 @@
+//CS300 Group #2
+//
+//Worked on by Tristan Gomez(started: 2/5/2020)
+
+struct serviceNode
+{
+	tString memNum;//to hold key to ensure proper data is fetched. 
```

Comment:
Why have this here? Isn't it already stored in service?

Reply:
I'm assuming it's to hold the key to ensure the proper data is fetched? With this being a hierarchy there are somethings that are off limits to other classes. I'm still not too sure